analysis_runner/cli_analysisrunner.py

```diff
@@ -35,6 +35,19 @@ def add_analysis_runner_args(parser=None) -> argparse.ArgumentParser:
 
     add_general_args(parser)
 
+    parser.add_argument(
+        '--container',
```

Comment:
Would `docker_image` be more precise? I.e. doesn't a container usually refer to a specific instance of a Docker image? (Or alternatively just `image`, to mirror the Hail Batch API.)

Reply:
Yeah I don't mind image, I don't like `docker` because any OCI compliant image works.